### DISCUSSION TITLE
More flexibility when changing file timestamp

### DISCUSSION THREAD

**Discussion by ags1234** [doublecmd/doublecmd#1106] [Suggestions]
### Description

I am missing this very cool option from Total Commander where the user can copy the timestamp from an item selected in the other panel.
If someone finds the time or interest to implement this, it would be great.

**Comment by Skif-off** [comment]
"Load from file under cursor" in the "Change attributes" window?
Total Commander has two buttons: "From target" and "From active panel (source)". It is logical to add _both_ buttons :)

**Comment by kakadu266** [comment]
It would also be important for me, I use the similar function of Total Commander a lot, but Double Commander is so much better, than TC that I would prefer to use it always and for everything.
And of course, it would be even more important to be able to use the Copy-Paste function in the Change attributes window.
